internal/guestrequest/types.go

```diff
@@ -66,7 +67,7 @@ type GuestRequest struct {
 }
 
 type NetworkModifyRequest struct {
-	AdapterInstanceId string      `json:"AdapterInstanceId,omitempty"`
-	RequestType       string      `json:"RequestType,omitempty"`
-	Settings          interface{} `json:"Settings,omitempty"`
+	AdapterId   string      `json:"AdapterId,omitempty"`
```

Comment:
@jhowardmsft - Are these v2 guest types? Is this safe to change without a version check?

Reply:
Let me add a method to fetch the right settings based on OS Version

Reply:
Done